haproxy.cfg

```diff
@@ -54,6 +52,8 @@ frontend router
   use_backend content-feed if { path_beg /content-feed }
   # All other apis should go to mastodon
   use_backend mastodon if { path_beg /api }
+  # Manifest endpoint
+  use_backend mastodon if { path_beg /manifest }
```

Comment:
Is there a reason to switch to mastodon manifest? Elk has it's own localized manifest files that we are currently serving, e.g. https://mozilla.social/manifest-en-US.webmanifest

Since manifests are a front-end / client thing, it makes sense that Elk should be the owner/server.

Reply:
This is the module that Elk uses to inject the manifests: https://github.com/mozillasocial/elk/blob/main/modules/pwa/i18n.ts

Reply:
currently the web manifest is broken in Elk. I can revert this change.